nrf_security/cmake/psa_crypto_config.cmake

```diff
@@ -236,6 +236,8 @@ kconfig_check_and_set_base_to_one(MBEDTLS_PSA_BUILTIN_ECC_SECP_R1_256)
 kconfig_check_and_set_base_to_one(MBEDTLS_PSA_BUILTIN_ECC_SECP_R1_384)
 kconfig_check_and_set_base_to_one(MBEDTLS_PSA_BUILTIN_ECC_SECP_R1_521)
 
+kconfig_check_and_set_base_to_one(PSA_CRYPTO_DRIVER_ENTROPY_ZEPHYR)
```

Comment:
Unrelated to the line but the commit message is wrong:
This allows us to get entropy on devices with nrf_cc3xx_platform support
i.e devices without cryptocell.

You want to say "without nrf_cc3xx_platform support" I assume.

Reply:
Thanks. Updated.